README.md

```diff
@@ -46,6 +46,44 @@ sure to install the new TALs using
 routinator init --force
 ```
 
+## Quick Start with Pre-built Packages
+
+### Disclaimer
+> These packages are provided on a best effort basis as a convenience for our community until such time as equivalent official operating system repository provided packages become available.
+
+Assuming you have a machine running a recent Debian or Ubuntu distribution, you
+can install Routinator from our [software package
+repository](https://packages.nlnetlabs.nl). To use this repository, add the line
+below that corresponds to your operating system to  your `/etc/apt/sources.list`
+or `/etc/apt/sources.list.d/`
+
+```bash
+deb [arch=amd64] https://packages.nlnetlabs.nl/linux/debian/ stretch main
+deb [arch=amd64] https://packages.nlnetlabs.nl/linux/debian/ buster main
+deb [arch=amd64] https://packages.nlnetlabs.nl/linux/ubuntu/ xenial main
+deb [arch=amd64] https://packages.nlnetlabs.nl/linux/ubuntu/ bionic main
+deb [arch=amd64] https://packages.nlnetlabs.nl/linux/ubuntu/ focal main
+```
+Then run the following commands.
+
+```bash
+sudo apt update && apt-get install -y gnupg2
+wget -qO- https://packages.nlnetlabs.nl/aptkey.asc | sudo apt-key add -
+sudo apt update
+```
+
+You can then install, enable and start Krill by running
+
+```bash
+sudo apt install routinator
+sudo routinator-init --accept-arin-rpa
```

Comment:
So far I always avoided providing a pastable snippet with the `--accept-arin-rpa` in it so that people are forced to at least consider reading the RPA. I think this would be good here, too.